drivers/hud_path_report/app/controllers/hud_path_report/cells_controller.rb

```diff
@@ -0,0 +1,35 @@
+###
+# Copyright 2016 - 2021 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module HudPathReport
+  class CellsController < HudPathReport::QuestionsController
+    before_action :set_report
+    before_action :set_question
+
+    def show
+      @cell = @report.valid_cell_name(params[:id])
+      @table = @report.valid_table_name(params[:table])
+      @clients = HudPathReport::Fy2020::DqClient.
+        joins(hud_reports_universe_members: { report_cell: :report_instance }).
+        merge(::HudReports::ReportCell.for_table(@table).for_cell(@cell)).
+        merge(::HudReports::ReportInstance.where(id: @report.id))
+      @name = "#{report_short_name} #{@question} #{@cell}"
```

Comment:
Do we need to sanitize this in any way?

Reply:
`report short name` comes from us, and `@question` and `@cell` are sanitized above... So, I _think_ we are OK... However, I did find that `valid_table_name` was not going to accept the tables with dashes in their names...